modules/ob-creating-a-build-with-oci-artifacts.adoc

```diff
@@ -0,0 +1,91 @@
+// This module is included in the following assembly:
+//
+// * work-with-builds/using-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="Creating-a-build-with-OCI-artifacts_{context}"]
+= Creating a build with OCI artifacts
+
+You can create a build using an Open Container Initiative(OCI) artifact as your source code. An OCI artifact, also known as a scratch image, contains only the source code and is not intended to run as a container. You can pull the OCI artifact from a container registry and extract its contents as source code to run the build.
+
+.Prerequisites
+
+* You have installed the {builds-operator} on the {ocp-product-title} cluster.
+* You have installed the `oc` command-line interface (CLI).
+* You have installed the link:https://console.redhat.com/openshift/downloads[`shp` CLI].
+
+.Procedure
+
+. Create a `Build` resource and apply it to the {ocp-product-title} cluster. See the following example configuration:
++
+[source,yaml]
```

Comment:
Can we add an ".Example" tag before the YAML to give a title to the code? It is as per our guidelines. Also, the YAML contains an oc apply -f command and YAML code, which does not look correct. So, I would suggest that you check this with your SME too. We should ideally separate the command and YAML code.